docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -2283,7 +2332,8 @@ \subsection{Display of Stake Pools in the Wallet}
 wallet should monitor the non-myopic rewards regularly. Should the stake pool
 become less favourable (by missing blocks, or even becoming inactive, or by
 changing its cost/margin), the wallet should notify the user, and ask them to
-consider changing their delegation.
+consider changing their delegation.\todo{Specify how many places the pool has to
+  drop in order to trigger this notification}
```

Comment:
Yeah, if it's about order, or absolute or relative change in the quality metric. Needs input from the researchers.